opentelemetry-sdk/src/export/logs/stdout.rs

```diff
@@ -0,0 +1,139 @@
+//! # Stdout Log Exporter
+//!
+//! The stdout [`LogExporter`] writes debug printed [`LogRecord`]s to its configured
```

Comment:
stdout has been moved to own create https://github.com/open-telemetry/opentelemetry-rust/pull/1027

I think its best for logs to also follow the same approach.